src/gui/errortabwidget.cpp

```diff
@@ -58,16 +58,15 @@ ErrorTabWidget::ErrorTabWidget(int driveDbId, bool generic, QWidget *parent)
 
     _resolveButton = new QPushButton(this);
     _resolveButton->setObjectName("resolveButton");
-    _resolveButton->setText(tr("Resolve"));
     _resolveButton->setVisible(false);
     MenuWidgetLite *menu = new MenuWidgetLite(_resolveButton, this);
 
-    _resolveConflictsAction = new QAction(tr("Conflicted item(s)"), this);
+    _resolveConflictsAction = new QAction(this);
```

Comment:
```suggestion
    _resolveConflictsAction = new QAction();
```

Reply:
Why? I don't think it is a good idea. Providing a parent to a QWidget ensure that the child will be automatically deleted when the parent is deleted.
`The ownership of action is not transferred to this [QWidget](https://doc.qt.io/qt-6/qwidget.html)` (https://doc.qt.io/qt-6/qwidget.html#addAction)

Reply:
You are absolutely right, thanks for the doc check: it is indeed true that ownership is not transferred to `menu` through `addAction`, so that the actions need a parent at the time they are created. My mistake.

In the context of my other comments (actions excepted), the widget `this` is often not the actual parent of the instantiated widget or layout. The parent is usually set later by some `Qt` setter. So,  it often happens that we set first an incorrect parent and then we set the correct one. I would prefer to skip the first step when it is sensible.

Reply:
Yes, you are totally right, it is the case at least for all `QLayout`. I'll fix it, thx.

Reply:
Done